lib/syskit/cli/log_runtime_archive_main.rb

```diff
@@ -63,19 +65,22 @@ def archive(root_dir, target_dir)
                 )
             end
 
-            desc "watch_transfer", "watches a dataset root folder \
-                                    and periodically performs transfer"
+            desc "watch_transfer SOURCE_DIR HOST PORT CERTIFICATE_PATH USER PASSWORD",
+                 "watches a dataset root folder and periodically performs transfer"
             option :period,
                    type: :numeric, default: 600, desc: "polling period in seconds"
             option :max_upload_rate_mbps,
                    type: :numeric, default: 1_000, desc: "max upload rate in Mbps"
+            option :implicit_ftps,
+                   type: :boolean, default: true,
```

Comment:
Actually, `false` would be a better default. `--implicit-ftps` is needed because of this 2.5/2.7 mixed situation, but won't be long term.